taskt/Core/Automation/Commands/Error Handling Commands/ThrowExceptionCommand.cs

```diff
@@ -1,38 +1,40 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Windows.Forms;
+using taskt.Core.Automation.Attributes.ClassAttributes;
 using taskt.UI.CustomControls;
 using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Error Handling Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to throw an exception error.")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to throw an exception error")]
-    [Attributes.ClassAttributes.ImplementationDescription("")]
+    [Group("Error Handling Commands")]
+    [Description("This command allows you to throw an exception error.")]
     public class ThrowExceptionCommand : ScriptCommand
     {
         public string ErrorMessage { get; set; }
         public string StackTrace { get; set; }
 
         public ThrowExceptionCommand()
         {
-            this.CommandName = "ThrowExceptionCommand";
-            this.SelectionName = "Throw Exception";
-            this.CommandEnabled = true;
-            this.CustomRendering = true;
+            CommandName = "ThrowExceptionCommand";
+            SelectionName = "Throw Exception";
+            CommandEnabled = true;
+            CustomRendering = true;
         }
+
         public override void RunCommand(object sender)
         {
-            throw new System.Exception();
+            throw new Exception();
```

Comment:
Wouldnt we want a way to attach a message or type/exception itself. We can save this improvement for the next sprint,

Reply:
There's a get exception message command for that.